examples/cmd/run.py

```diff
@@ -27,12 +29,60 @@ def save_mp3_file(wav, index):
     logger.info(f"Audio saved to {mp3_filename}")
 
 
-def main(texts: List[str], spk: Optional[str] = None, stream=False):
+def ndarray_to_tensor(audio: np.ndarray) -> torch.Tensor:
```

Comment:
Why need this function? It's not used at anywhere.

Reply:
@fumiama yep~, done,  remove, some example code don't commit